pyproject.toml

```diff
@@ -104,7 +104,35 @@ type = [
     { directory = "misc", name = "Misc", showcontent = false },
 ]
 
-[tool.setuptools_scm]
+# [tool.setuptools_scm]
```

Comment:
This seems odd - either we don't need setuptools_scm config at the root level, or this has been commented out in error.